docusaurus-docs/community/contribute/local-setup-ui.md

```diff
@@ -9,46 +9,62 @@ To get started with conda-store-ui development, there are a couple of options. T
 
 ## Pre-requisites
 
-We use [Docker Compose](https://docs.docker.com/compose/) to set up the infrastructure before starting ensure that you have Docker compose installed. If you need to install docker-compose, please see their [installation documentation](https://docs.docker.com/compose/install/)
+This document assumes that you have the following tools set up on your computer:
 
-1. Fork and clone the [conda-store-ui](https://github.com/conda-incubator/conda-store-ui.git) repository.
-2. Copy `.env.example` to `.env`. All default settings should work, but if you want to test against a different version of conda-store-server, you can specify it in the `.env` file by setting the `CONDA_STORE_SERVER_VERSION` variable to the desired version
+- Docker Compose (which requires Docker)
+- Git
 
-## Docker (Recommended)
+You also need to have:
 
-Once you have a [local copy of the `conda-store` repository](community/contribute/contribute-code#setup-for-local-development)
+- A GitHub.com account
 
-Running conda-store-ui in Docker is the simplest way to set up your local development environment.
+Once you have all of those things, then you need to:
 
-1. Run `yarn run start:docker` to start the entire development stack.
-Open your local browser and go to [http://localhost:8000](http://localhost:8000) so see conda-store-ui.
-3. You can then log in using the default username of `username` and default password of `password`.
+1. [Fork the conda-store-ui repository](https://github.com/conda-incubator/conda-store-ui/fork)
+2. `git clone` the fork to your computer.
 
-**Note:** Hot reloading is enabled, so when you make changes to source files, your browser will reload and reflect the changes.
+## Basic Option (recommended): Run the UI in Docker
+
+For basic local development on the UI, running conda-store-ui in Docker is the simplest way to get started.
+
+1. Change your working directory to the project root: `cd conda-store-ui`
+2. Optional. Set environment variables. Copy `.env.example` to `.env`. All default settings should work, but if you want to test against a different version of conda-store-server, you can specify it in the `.env` file by setting the `CONDA_STORE_SERVER_VERSION` variable to the desired version. If you skip this step, it will be done automatically for you.
+3. Run `docker compose --profile local-dev up --build` to start the entire development stack. This step may take a few minutes the first time you run it.
+4. Open your local browser and go to [http://localhost:8000](http://localhost:8000) to see conda-store-ui.
+5. You can then log in using the default username of `username` and default password of `password`.
+
+**Note:** Hot reloading is enabled, so when you make changes to source files (i.e., files under the conda-store-ui/src/ directory), your browser will reload and reflect the changes.
```

Comment:
Since this is in the docs, let's use admonitions

```suggestion
:::note 

Hot reloading is enabled, so when you make changes to source files (i.e., files under the conda-store-ui/src/ directory), your browser will reload and reflect the changes.

:::
```